app.py

```diff
@@ -56,7 +57,8 @@ def set_session_permanency():
 
 
 # Register Markdown engine
-Misaka(app)
+# Misaka(app, None, hard_wrap=True)
+app.add_template_filter(markdown)
```

Comment:
I think you can remove `from flask_mikasa import Mikasa` and the overall dependency now, right?

Reply:
had forgot to save the app.py file, sigh. Update inc.